### ISSUE TITLE
Errors running in DNN 09.04.01

### ISSUE THREAD

**Issue by david-poindexter** [DNNCommunity/DNN.IFrame#21]
## To Reproduce
1. Install **DNN.IFrame 08.01.01** in **DNN 09.04.01**
2. Add **IFrame** module to a page
3. **Edit Options** and **Update**
4. See errors on-page and in **MANAGE > Admin Logs**

## Expected behavior
No errors.

## Screenshots
![image](https://user-images.githubusercontent.com/4568451/66690473-2a2a3180-ec5e-11e9-95f0-d7a56ff8205d.png)

![image](https://user-images.githubusercontent.com/4568451/66690394-ab34f900-ec5d-11e9-8772-47f8036389fa.png)

![image](https://user-images.githubusercontent.com/4568451/66690398-b851e800-ec5d-11e9-877f-e8f028d4da8e.png)

### Error Number 1
```
10/11/2019 19:29:59
General Exception
host 
DNN 09.04.01 
AbsoluteURL /Default.aspx DefaultDataProvider DotNetNuke.Data.SqlDataProvi 
AbsoluteURL:/Default.aspx

DefaultDataProvider:DotNetNuke.Data.SqlDataProvider, DotNetNuke

ExceptionGUID:cfaad374-9d68-40b1-bafb-3045ba10a6ea

AssemblyVersion:

PortalId:-1

UserId:-1

TabId:-1

RawUrl:

Referrer:

UserAgent:

ExceptionHash:11leUp3OBtTle2lQkWbW7E1Kj40=

Message:Invalid column name 'ModuleController'. Invalid column name 'ModuleInfo'. Invalid column name 'PortalController'. Invalid column name 'PortalInfo'.

StackTrace:

   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   at System.Data.SqlClient.SqlDataReader.get_MetaData()
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   at System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   at System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   at System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   at PetaPoco.Database.<ExecuteReader>d__149`1.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at DotNetNuke.Common.Utilities.DataCache.GetCachedDataFromRuntimeCache(CacheItemArgs cacheItemArgs, CacheItemExpiredCallback cacheItemExpired)
InnerMessage:

InnerStackTrace:

Source:.Net SqlClient Data Provider

FileName:

FileLineNumber:0

FileColumnNumber:0

Method:
```

### Error Number 2
```
10/11/2019 19:29:59
Module Load Exception
host 
DNN 09.04.01 
ModuleId 380 ModuleDefId 122 FriendlyName IFrame ModuleControlSource Deskt 
ModuleId:380

ModuleDefId:122

FriendlyName:IFrame

ModuleControlSource:DesktopModules/IFrame/IFrame.ascx

AbsoluteURL:/Default.aspx

DefaultDataProvider:DotNetNuke.Data.SqlDataProvider, DotNetNuke

ExceptionGUID:137a9ff5-0ec8-4938-a477-daccc3772a41

AssemblyVersion:9.4.1

PortalId:0

UserId:1

TabId:33

RawUrl:/iFrame-Test

Referrer:http://dnn941.loc/iFrame-Test

UserAgent:Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36

ExceptionHash:TwWAxm98R9QTkOUFMYeFJzEpL7E=

Message:Value cannot be null. Parameter name: source

StackTrace:

InnerMessage:Value cannot be null. Parameter name: source

InnerStackTrace:

   at System.Linq.Enumerable.Count[TSource](IEnumerable`1 source)
   at DotNetNuke.Modules.IFrame.UI.IFrame.get_Source()
   at DotNetNuke.Modules.IFrame.UI.IFrame.Page_Load(Object sender, EventArgs e)
Source:

FileName:

FileLineNumber:0

FileColumnNumber:0

Method:
```

## Additional context
Fresh install of DNN 09.04.01

**Comment by MaiklT** [comment]
Fixed in version 08.01.02

**Comment by valadas** [comment]
Awesome! Just in time for another dnn upgrade I needed it for. Thanks @MaiklT
